### ISSUE TITLE
Screen Reader Support Specification Adaptation - Avatar Group

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2662]
**Is your feature request related to a problem? Please describe.**
We should analyse the existing screen reader support implementation, along with the UI5 SR Specification itself and Aria Best Practices and adapt them in a way that fit the ui5 web components.

**Describe the solution you'd like**
Analyse, create and discuss a proposal specification with @elenastoyanovaa for the avatar group component.

After the discussion, the approved screen reader support implementation proposal, should be documented in the accessibility status wiki of the component.

**Comment by elenastoyanovaa** [comment]
The specification proposal was created and the implementation will be done in https://github.com/SAP/ui5-webcomponents/issues/2745.
